src/main.c

```diff
@@ -1349,7 +1349,7 @@ main(int argc, char **argv)
   pthread_sigmask(SIG_UNBLOCK, &set, NULL);
 
   tvhlog(LOG_NOTICE, LS_START, "HTS Tvheadend version %s started, "
-         "running as PID:%d UID:%d GID:%d, CWD:%s CNF:%s",
+         "running as PID:%u UID:%u GID:%u, CWD:%s CNF:%s",
```

Comment:
According to https://www.gnu.org/software/libc/manual/html_node/Process-Identification.html `getpid()` returns an `int` and not an `unsigned int`?